### PR TITLE
Gracefully handle failure in loading model.pkl from disk

### PR DESCRIPTION
**Pull request by gaugup**
There could be a situation where pickle.load() cannot load the model. In such scenarios, the load() of the RAIInsights will fail. Adding additional logic to cover this scenario.

### CODE REVIEW

File: responsibleai/responsibleai/rai_insights/rai_insights.py [hunk 1]
```diff
@@ -714,8 +714,11 @@ def _load_model(inst, path):
             inst.__dict__[_MODEL] = serializer.load(top_dir)
         else:
             inst.__dict__['_' + _SERIALIZER] = None
-            with open(top_dir / _MODEL_PKL, 'rb') as file:
-                inst.__dict__[_MODEL] = pickle.load(file)
+            try:
+                with open(top_dir / _MODEL_PKL, 'rb') as file:
+                    inst.__dict__[_MODEL] = pickle.load(file)
+            except Exception:
+                inst.__dict__[_MODEL] = None
```
Comment (riedgar-ms) [inline responsibleai/responsibleai/rai_insights/rai_insights.py:721]: Perhaps log a warning?

**Review by imatiach-msft — APPROVED** [review]

File: responsibleai/responsibleai/rai_insights/rai_insights.py [hunk 2]
```diff
@@ -714,8 +714,11 @@ def _load_model(inst, path):
             inst.__dict__[_MODEL] = serializer.load(top_dir)
         else:
             inst.__dict__['_' + _SERIALIZER] = None
-            with open(top_dir / _MODEL_PKL, 'rb') as file:
-                inst.__dict__[_MODEL] = pickle.load(file)
+            try:
```
Comment (imatiach-msft) [inline responsibleai/responsibleai/rai_insights/rai_insights.py:717]: not necessary but might be interesting to add a test for this scenario

File: responsibleai/responsibleai/rai_insights/rai_insights.py (hunk 1 above)
Comment (gaugup) [reply responsibleai/responsibleai/rai_insights/rai_insights.py:723]: Added the requested warning.

File: responsibleai/responsibleai/rai_insights/rai_insights.py (hunk 2 above)
Comment (gaugup) [reply responsibleai/responsibleai/rai_insights/rai_insights.py:715]: Added the test case.

**Pull request merged**